tests/codegen/min-function-alignment.rs

```diff
@@ -0,0 +1,30 @@
+//@ compile-flags: -C no-prepopulate-passes -Z mir-opt-level=0 -Zmin-function-alignment=16
```

Comment:
here we set `-Zmin-function-alignment=16`, which if `16` is interpreted as the argument to `2.pow(arg)`, would mean the alignment should be 65536? so the earlier documentation can't be correct.